bezel-cfgs/amidar.cfg

```diff
@@ -0,0 +1,12 @@
+# 1941 Counter Attack Overlay for Retroarch with 4:3 Display
```

Comment:
Could you please update the comment with the actual game name and you user name?

Reply:
I just got it that you meant my PR comment rather than the commit message from my repo. 🥴
I added the game name now. Which user name do you mean? Github or Retropie forum?

Reply:
@taranchul no, I meant in both `bezel-cfgs/amidar.cfg` and `bezel-cfgs/apb.cfg`. The first line in those files is a comment with the name of the game, you copy-pasted the one from 1941 but forgot to change the comment ;)
Regarding the user name, use whatever you like.

Reply:
Ahh, the stumbling stones of communication. Done. :)